integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/cases/MppProjectTest.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package test
+
+import kotlinx.atomicfu.gradle.plugin.test.framework.checker.*
+import kotlinx.atomicfu.gradle.plugin.test.framework.runner.*
+import kotlin.test.Test
+
+class MppProjectTest {
+    private val mppSample: GradleBuild = createGradleBuildFromSources("mpp-sample")
+
+    @Test
+    fun testMppJvm1() {
```

Comment:
Could you please use more verbose names (instead of `1` and `2`)?

Reply:
Done